src/scripts/Selector/SelectorLink.js

```diff
@@ -30,24 +30,24 @@ export default class SelectorLink extends Selector {
 	async _getData(parentElement) {
 		const elements = this.getDataElements(parentElement);
 		if (!this.multiple && !elements.length) {
-			return [{ [this.id]: null }];
+			return [{ [this.uuid]: null }];
```

Comment:
id! You should not use uuids as extracted data keys unless you transform them to ids later.
I'm ok with them in auxiliary keys like XXX-href, but they have to be removed from items afterwards.